papers/yellow/block.tex

```diff
@@ -0,0 +1,67 @@
+\section{Ergo Block Structure}
+Ergo block consists of 4 parts:
+
+\begin{itemize}
+    \item{\em Header } - minimal amount of data, required to synchronize the chain and check PoW correctness.
+    Also contains hashes of other sections.
+    \item{\em BlockTransactions } - sequence of transactions, included in this block.
+    \item{\em ADProofs } - proofs for transactions, included into BlockTransactions.
```

Comment:
"proofs for transactions included into the corresponding BlockTransactions section of a block."